docs/workflows-setup.md

```diff
@@ -0,0 +1,53 @@
+---
+id: workflows-setup
+title: Workflows setup
+sidebar_label: Workflows setup
+---
+
+Workflows automatically build and test your code with
+BuildBuddy whenever a commit is pushed to your GitHub repo.
+
+When combined with GitHub's branch protection rules, workflows can help prevent
+unwanted code (that doesn't build or pass tests) from being merged into the main branch.
+
+Best of all, workflows let you run any Bazel commands you would like,
+so you can leverage all the same BuildBuddy features that you get when
+running Bazel locally, like the results UI, remote caching, remote execution.
+
+## Enable workflows for a repo
+
+### 1. Link GitHub account {#link-github-account}
```

Comment:
are those anchors needed? i feel like docusaurus should generate them for titles automatically

Reply:
Sometimes I do this to decouple the title of the section from the link, so that the title can be changed without breaking links to that section. But I'm not really consistent about it, so may as well just remove for now.